cmd/argo/commands/clustertemplate/list.go

```diff
@@ -22,6 +22,15 @@ func NewListCommand() *cobra.Command {
 	command := &cobra.Command{
 		Use:   "list",
 		Short: "list cluster workflow templates",
+		Example: `# List Cluster Workflow Templates:
+  argo cluster-template list
+	
+# List Cluster Workflow Templates in "wide" output format:
+  argo cluster-template list --output wide
+  
+# List Cluster Workflow Templates in "name" output format:
```

Comment:
Same as above, this should be more specific. I believe `name` lists names only, but I'm not sure, haven't tried it

Reply:
@agilgur5 I have made the modifications and tried to be more specific. Please have a look at it once and also the windows unit tests are failing but I am not able to understand why !! Please throw some light on it as well !!

Reply:
The Windows tests aren't required to pass CI. They're new and still being worked on